$MirOS: ports/games/bsdgames/patches/patch-rogue_score_c,v 1.1.7.1 2005/03/18 15:46:07 tg Exp $
--- rogue/score.c.orig	Thu Jan  1 00:00:00 1970
+++ rogue/score.c	Thu Feb 22 21:51:08 2007
@@ -125,7 +125,7 @@ killed_by(const object *monster, short o
 	} else {
 		messagef(0, "%s", message_buf);
 	}
-	messagef(0, "");
+	messagef(0, NULL);
 	put_scores(monster, other);
 }
 
@@ -146,8 +146,8 @@ win(void)
 	mvaddstr(17, 11, "Congratulations,  you have  been admitted  to  the");
 	mvaddstr(18, 11, "Fighters' Guild.   You return home,  sell all your");
 	mvaddstr(19, 11, "treasures at great profit and retire into comfort.");
-	messagef(0, "");
-	messagef(0, "");
+	messagef(0, NULL);
+	messagef(0, NULL);
 	id_all();
 	sell_pack();
 	put_scores((object *) 0, WIN);
@@ -428,9 +428,9 @@ put_scores(const object *monster, short 
 		}
 	}
 	refresh();
-	messagef(0, "");
+	messagef(0, NULL);
 	if (dopause) {
-		messagef(0, "");
+		messagef(0, NULL);
 	}
 	clean_up("");
 }
@@ -527,7 +527,7 @@ sell_pack(void)
 	if (rogue.gold > MAX_GOLD) {
 		rogue.gold = MAX_GOLD;
 	}
-	messagef(0, "");
+	messagef(0, NULL);
 }
 
 int
@@ -645,6 +645,6 @@ void
 sf_error(void)
 {
 	md_lock(0);
-	messagef(1, "");
+	messagef(1, NULL);
 	clean_up("sorry, score file is out of order");
 }
